types/office-js-preview/index.d.ts

```diff
@@ -63740,6 +63864,16 @@ declare namespace Excel {
              * [Api set: ExcelApi 1.1]
              */
             rowIndex?: boolean;
+            /**
+             *
+             * For EACH ITEM in the collection: Represents if ALL the cells would be saved as an array formula
+            Returns true if ALL cells would be saved as an array, or false if ALL cells would NOT be saved as an array formula.
+            Returns null if there are cells both which would be saved as an array formula and would not be
```

Comment:
This seems incomplete.

Also, won't this all end up being on one line in the docs?